scripts/ec2_deploy_teardown/tearDown.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+echo "Obtaining VPC Details"
+VPC=`aws ec2 describe-vpcs --region ${CLUSTER_REGION} --filters "Name=tag:Name,Values=${clusterID}*" --query "Vpcs[0].VpcId" --output text` 
+for (( i=0; i<5; i++ )) do
+    if [[ $VPC =~ "vpc-" ]]; then
+      echo "VPC "${VPC}" is available"
+      break
+    fi
+    sleep 5
+done
+
+echo "Obtaining security group"
+SECGRP=`aws ec2 describe-security-groups --region ${CLUSTER_REGION} --filters Name=group-name,Values=pipeline-sg --query SecurityGroups[0].GroupId --output text`
+for (( i=0; i<5; i++ )) do
+    if [[ $SECGRP =~ "sg-" ]]; then
+      echo "SECGRP "${SECGRP}" identified"
+      break
+    fi
+    sleep 5
+done
+
+echo "Obtaining EC2 micro-t2 instance"
+EC2=`aws ec2 describe-instances  --region ${CLUSTER_REGION} --filters Name=key-name,Values=pipelineKey Name=instance-state-name,Values=running --query Reservations[0].Instances[0].InstanceId --output text`
+for (( i=0; i<5; i++ )) do
+    if [[ $EC2 =~ "i-" ]]; then
+      echo "EC2 "${EC2}" identified"
+      break
+    fi
+    sleep 5
+done
+
+echo "Obtaining address allocation details"
+ELIP=`aws ec2 describe-addresses --filters Name=instance-id,Values=$EC2 --region ${CLUSTER_REGION} --query Addresses[0].AllocationId --output text`
+for (( i=0; i<5; i++ )) do
+    if [[ $ELIP =~ "eipalloc-" ]]; then
+      echo "Address "${ELIP}" identified"
+      break
+    fi
+    sleep 5
+done
+
+echo "Obtaining address association details"
+ASSOC=`aws ec2 describe-addresses --filters Name=instance-id,Values=$EC2 --region ${CLUSTER_REGION} --query Addresses[0].AssociationId --output text`
+for (( i=0; i<5; i++ )) do
+    if [[ $ASSOC =~ "eipassoc-" ]]; then
+      echo "Address "${ASSOC}" identified"
+      break
+    fi
+    sleep 5
+done
+
+echo "Getting image"
+AMI=`aws ec2 describe-images --filters "Name=is-public,Values=true" "Name=description,Values=Provided by Red Hat*" "Name=name,Values=RHEL-8.2.0_HVM-*-x86_64*Hourly2-GP2" --region ${CLUSTER_REGION} --output text --query 'Images[0].ImageId'`
```

Comment:
not required, you don't need to know the AMI for teardown. You are not using it anywhere.

Reply:
Done